osu.Game.Rulesets.Catch.Tests/TestSceneLegacyBeatmapSkin.cs

```diff
@@ -101,15 +101,10 @@ public void TestBeatmapHyperDashColoursOverride(bool useBeatmapSkin)
             AddAssert("is custom hyper dash fruit colours", () => ((CatchExposedPlayer)TestPlayer).UsableHyperDashFruitColour == TestSkin.HYPER_DASH_FRUIT_COLOUR);
         }
 
-        protected override ExposedPlayer CreateTestPlayer(bool userHasCustomColours) => new CatchExposedPlayer(userHasCustomColours);
+        protected override ExposedPlayer CreateTestPlayer() => new CatchExposedPlayer();
```

Comment:
Can you remind me how this still works / where this logic has been moved to?

Reply:
Check `CreateTestPlayer` logic in base test scene.